File: programming.in.th/3-11_(three-eleven)/three_eleven.py
number = input()

mod_by_3 , mod_by_11 = 0 , 0 
for e in number :
    mod_by_3 = (mod_by_3 * 10 + int(e)) % 3
    mod_by_11 = (mod_by_11 * 10 + int(e)) % 11

print(f"{mod_by_3} {mod_by_11}")


""" 

   input = 11 
   first round 
   mod_by_3 = ( 0 * 10 + 1)  % 3   :=> 1
   mod_by_11 = ( 0 * 10 + 1) % 11 :=> 1

   sec round 
   mod_by_3 = ( 1 * 10 + 1 ) % 3  :=> 2
   mod_by_11 = ( 1 * 10 + 1 ) % 11 :=> 0


"""

File: programming.in.th/3-11_(three-eleven)/three_eleven.java
    import java.io.BufferedReader;
    import java.io.IOException;
    import java.io.InputStreamReader;

    public class three_eleven {
        
        public static void main(String[] args) throws IOException {
            
            BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
            String input = reader.readLine();

            int n , mod_by_3 = 0 , mod_by_11 = 0;
            
            for (char digit : input.toCharArray())
            {
                n = digit - '0';
                mod_by_3= (mod_by_3 * 10 + n) % 3;
                mod_by_11 = (mod_by_11 * 10 + n) % 11;
            }

            System.out.println(String.format("%d %d",mod_by_3 , mod_by_11));
        
        }
        
    }


